markdown/advanced/03-kolmogorov_equations.md

```diff
@@ -0,0 +1,1214 @@
+Current loss is: 103.60637
+Current loss is: 103.60637
+Current loss is: 103.60635
+Current loss is: 103.60635
+Current loss is: 103.60635
+Current loss is: 103.60635
+Current loss is: 103.60635
+Current loss is: 103.606346
+Current loss is: 103.606346
+Current loss is: 103.606346
+Current loss is: 103.606346
+Current loss is: 103.606346
+Current loss is: 103.606346
+Current loss is: 103.606346
+Current loss is: 103.606346
+Current loss is: 103.60637
+Current loss is: 103.60637
+Current loss is: 103.606384
+Current loss is: 103.6064
+Current loss is: 103.60644
+Current loss is: 103.606514
+Current loss is: 103.60663
+Current loss is: 103.60683
+Current loss is: 103.607185
+Current loss is: 103.60782
+Current loss is: 103.608955
+Current loss is: 103.61094
+Current loss is: 103.614624
+Current loss is: 103.62094
+Current loss is: 103.63288
+Current loss is: 103.65198
+Current loss is: 103.68764
+Current loss is: 103.73291
+Current loss is: 103.80574
+Current loss is: 103.83829
+Current loss is: 103.850174
+Current loss is: 103.74804
+Current loss is: 103.64673
+Current loss is: 103.607574
+Current loss is: 103.65114
+Current loss is: 103.71751
+Current loss is: 103.71883
+Current loss is: 103.67134
+Current loss is: 103.61534
+Current loss is: 103.612946
+Current loss is: 103.65107
+Current loss is: 103.67022
+Current loss is: 103.651085
+Current loss is: 103.61496
+Current loss is: 103.609024
+Current loss is: 103.63142
+Current loss is: 103.64416
+Current loss is: 103.63231
+Current loss is: 103.611
+Current loss is: 103.607925
+Current loss is: 103.62147
+Current loss is: 103.62928
+Current loss is: 103.62204
+Current loss is: 103.60921
+Current loss is: 103.607216
+Current loss is: 103.61528
+Current loss is: 103.61992
+Current loss is: 103.61536
+Current loss is: 103.607864
+Current loss is: 103.60691
+Current loss is: 103.611755
+Current loss is: 103.614555
+Current loss is: 103.611916
+Current loss is: 103.6074
+Current loss is: 103.606544
+Current loss is: 103.60926
+Current loss is: 103.611145
+Current loss is: 103.60984
+Current loss is: 103.60714
+Current loss is: 103.606346
+Current loss is: 103.60783
+Current loss is: 103.60913
+Current loss is: 103.608604
+Current loss is: 103.60702
+Current loss is: 103.606255
+Current loss is: 103.606926
+Current loss is: 103.60783
+Current loss is: 103.60783
+Current loss is: 103.60693
+Current loss is: 103.60628
+Current loss is: 103.60644
+Current loss is: 103.60701
+Current loss is: 103.60724
+Current loss is: 103.60685
+Current loss is: 103.60635
+Current loss is: 103.606255
+Current loss is: 103.60651
+Current loss is: 103.60679
+Current loss is: 103.60672
+Current loss is: 103.60643
+Current loss is: 103.60624
+Current loss is: 103.60627
+Current loss is: 103.606445
+Current loss is: 103.60654
+Current loss is: 103.606445
+Current loss is: 103.606285
+Current loss is: 103.6062
+Current loss is: 103.60625
+Current loss is: 103.606346
+Current loss is: 103.60638
+Current loss is: 103.606316
```

Comment:
@ashutosh-b-b only goes to 100

Reply:
In paper it was expected to attain a minimum value...

Reply:
yes, but that still seems a little high?

Reply:
Oh. But It doesnt go down past that. I tried a really long train as well. And since the analytical error was converging I thought this minima should be fine.